File: WinPaletter/Store/Store_ThemeLicense.vb
Public Class Store_ThemeLicense

    Private Sub Store_ThemeLicense_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        LoadLanguage
        ApplyStyle(Me)
        TextBox1.Font = My.Application.ConsoleFontLarge
        Icon = LicenseForm.Icon
        My.Computer.Audio.PlaySystemSound(Media.SystemSounds.Exclamation)
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        DialogResult = DialogResult.OK
        Close()
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        DialogResult = DialogResult.Cancel
        Close()
    End Sub

End Class

File: WinPaletter/Store/Store_ThemeLicense.cs
using System;
using System.Media;
using System.Windows.Forms;

namespace WinPaletter
{
    /// <summary>
    /// Theme license form for displaying the license of a theme.
    /// </summary>
    public partial class Store_ThemeLicense
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="Store_ThemeLicense"/> class.
        /// </summary>
        public Store_ThemeLicense()
        {
            InitializeComponent();
        }

        private void Store_ThemeLicense_Load(object sender, EventArgs e)
        {
            this.Localize();
            ApplyStyle(this);
            TextBox1.Font = Fonts.ConsoleLarge;

            CustomSystemSounds.Exclamation.Play();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            DialogResult = DialogResult.OK;
            Close();
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            DialogResult = DialogResult.Cancel;
            Close();
        }

    }
}